src/components/form/file_picker/_file_picker.scss

```
@@ -115,7 +115,8 @@
 
   // make normal sized prompt look like placeholder
   .euiFilePicker:not(.euiFilePicker--large):not(.euiFilePicker-hasFiles) & {
-    color: $euiColorMediumShade;
+    color: $euiColorPrimaryText;
+    margin-top: calc(#{$euiSizeXS} / -2);
```

This shouldn't need to be a calc, should it?

```suggestion
    margin-top: $euiSizeXS} / -2;
```

Note: I'm being a stickler about this only because of our transition to Emotion - Sass will automatically **remove** CSS `calc()` if it can figure out the math on its own, but Emotion **does not do this**, so we end up with a bunch of CSS like `calc(2px)` in our CSS when things like this gets converted.